resources/references/core-reference/administration-reference/utils.md

```diff
@@ -0,0 +1,109 @@
+# Utils reference
+
+This is an overview of all the utility functions bound to the shopware global object.
+Utility functions provide many useful shortcuts for common tasks, see how to use them in your plugin [here](./../../../../guides/plugins/plugins/administration/using-utils.md).
+
+## General functions
```

Comment:
Some Descriptions end with a punctuation, some not 
Please straighten this to one format, preferrably without punctuations

Reply:
Agreed. I left two examples as suggestions, but most line's format is incorrect.

@NiklasLimberg Please fix that in all lines necessary.